src/QMCTools/LCAOHDFParser.cpp

```diff
@@ -326,17 +326,9 @@ void LCAOHDFParser::getMO(const std::string& fname)
   char name[72];
   sprintf(name, "%s", "/Super_Twist/eigenset_0");
   setname = name;
-  if (!hin.readEntry(CartMat, setname))
-  {
-    setname = "SPOSet::putFromH5 Missing " + setname + " from HDF5 File.";
-    APP_ABORT(setname.c_str());
-  }
+  hin.read(CartMat, setname);
```

Comment:
Fine with this change. Using the built-in message in read/write doesn't reduce infomation to users.